packages/babel-plugin-transform-regenerator/test/arrow-inside-generator.js

```diff
@@ -0,0 +1,19 @@
+const assert = require("assert");
+const babel = require("babel-core");
+
+test("Doesn't throw with arrow function inside generator", function () {
```

Comment:
@loganfsmyth thanks! Initially, I created it for 6.0 as additional for https://github.com/babel/babel/pull/5567 and needed to prevent running code (to prevent error with arrow functions with node < 4).

I could change base branch to 6.0 or just update it to run with `exec.js`.